frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx

```diff
@@ -339,6 +447,15 @@ export function InstallSubmarinerForm(props: { availableClusters: Cluster[] }) {
                         isRequired: true,
                         options: [...availableClusters.map((c) => ({ id: c.displayName!, value: c.displayName! }))],
                     },
+                    {
+                        id: 'globalist-enable',
+                        type: 'Checkbox',
+                        label: t('Enable Globalnet'),
```

Comment:
Instead of using "Enable Globalnet" as a heading (shown in RED box below). Shall we use "Globalnet Status" for the heading?

![Screenshot from 2022-03-03 12-50-00](https://user-images.githubusercontent.com/6574656/156516116-16a9701f-3f1b-47b2-ad6b-72a330f47563.png)

Reply:
`Globalnet Status` will not make sense for the first cluster when we're setting the value. Another option could be a more neutral `Globalnet Enabled` as that would work both for setting option as well as status.

Reply:
Currently in the image, "Enable Globalnet" is repeated as you can see - one is the heading and the other is the checkbox. I want the HEADING to be updated to "Globalnet Status" while retaining the "Enable Globalnet" checkbox as-is. 

In this manner, when subsequent clusters are added to the ClusterSet, the Heading will be "Globalnet Status" and the checkbox will be grayed out (i.e., only readable).

Reply:
Aha, got it. I missed the point. So it would work as `Globalnet Status` for heading and just `Enabled` for checkbox.

Reply:
That's fine too.

Reply:
For now its only "Enable Globalnet", but in future when we expose additional Globalnet configuration (like defaultGlobalnetCIDR, defaultGlobalnetClusterSize etc) it can go under the same section.

Reply:
How about **Globalnet** or **Globalnet Settings** for the heading?